tcms/templates/run/get_case_runs.html

```diff
@@ -23,33 +23,34 @@
 				{% if perms.testruns.delete_testcaserun %}
                                 <li><a href="#" title="Remove selected cases form this test run" data-param="{{ test_run.run_id }}" class="removeBlue9 js-del-case">{% trans "Remove" %}</a></li>
 				{% endif %}
+
 				{% if perms.testruns.change_testcaserun %}
-                                <li><a href="#" title="Update the IDLE case runs to newest case text" href="javascript:void(0)" data-param="{% url 'testruns-update_case_run_text' test_run.pk %}" class="updateBlue9 js-update-case">{% trans "Update" %}</a></li>
-				{% endif %}
-				{% if perms.testruns.change_testcaserun %}
-                                <li><a href="#" title="Assignee this case(s) to other people" class="assigneeBlue9 js-change-assignee">{% trans "Assignee" %}</a></li>
+	                <li><a href="#" title="Update the IDLE case runs to newest case text" href="javascript:void(0)" data-param="{% url 'testruns-update_case_run_text' test_run.pk %}" class="updateBlue9 js-update-case">{% trans "Update" %}</a></li>
+	                <li><a href="#" title="Assignee this case(s) to other people" class="assigneeBlue9 js-change-assignee">{% trans "Assignee" %}</a></li>
 				{% endif %}
 			</ul>
 		</div>
 		<div class="btnBlueCaserun">
                     <span>{% trans "Status" %}</span>
-			<ul class="statusOptions">
-				{% for tcrs in test_status %}
-				<li><a value="{{ tcrs.pk }}" href="#" class="{{ tcrs|lower }}Blue9">{{ tcrs }}</a></li>
-				{% endfor %}
-			</ul>
+			{% if perms.testruns.change_testcaserun %}
+				<ul class="statusOptions">
+					{% for tcrs in test_status %}
+					<li><a value="{{ tcrs.pk }}" href="#" class="{{ tcrs|lower }}Blue9">{{ tcrs }}</a></li>
+					{% endfor %}
+				</ul>
+			{% endif %}
 		</div>
 		<div class="btnBlueCaserun">
                     <span>{% trans "Bugs" %}</span>
 			<ul>
-                            <li><a href="javascript:void(0);" class="addBlue9 js-add-bugs">{% trans "Add" %}</a></li>
-                            <li><a href="javascript:void(0);" class="removeBlue9 js-remove-bugs">{% trans "Remove" %}</a></li>
+                        <li><a href="javascript:void(0);" class="addBlue9 js-add-bugs">{% trans "Add" %}</a></li>
+                        <li><a href="javascript:void(0);" class="removeBlue9 js-remove-bugs">{% trans "Remove" %}</a></li>
```

Comment:
Do not change spaces if you are not changing anything else. It makes the PR harder to read/review and doesn't bring anything useful.

Reply:
Not resolved, revert white-space changes if nothing else changed.